DeviceDiagnostics/DeviceDiagnosticsPlugin.json

```diff
@@ -5,7 +5,7 @@
       "callsign": "org.rdk.DeviceDiagnostics",
       "locator": "libWPEFrameworkDeviceDiagnostics.so",
       "status": "production",
-      "description": "The `DeviceDiagnostics` plugin provides an interface for the WebPA service for this device only. It allows you to get device configurations.",
+      "description": "The `DeviceDiagnostics` plugin provides an interface for the WebPA service for this device only. It allows you to get device configurations and AV decoder status.",
```

Comment:
Pls change this to something like "The DeviceDiagnostics Plugin provides additional diagnostics information. This includes device configuration and AV decoder status"

Reply:
Modified the main description as per suggestion